core/org.openjdk.jmc.flightrecorder.serializers/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Name: Common Plug-in
+Bundle-SymbolicName: org.openjdk.jmc.flightrecorder.serializers;singleton:=true
+Bundle-Version: 8.1.0.qualifier
+Bundle-Vendor: Oracle Corporation
+Export-Package: org.openjdk.jmc.flightrecorder.serializers,
+ org.openjdk.jmc.flightrecorder.serializers.json,
+ org.openjdk.jmc.flightrecorder.serializers.dot
+Automatic-Module-Name: org.openjdk.jmc.flightrecorder.serializers
+Require-Bundle: org.lz4.lz4-java
```

Comment:
Looks like it could use `org.openjdk.jmc.flightrecorder` in the require-bundle list? Otherwise when trying to use the Flamegraph or Graphview I end up with lots of exceptions such as:

```
Exception in thread "FlameGraphModelCalculation-0" java.lang.NoClassDefFoundError: org/openjdk/jmc/flightrecorder/stacktrace/tree/StacktraceTreeModel
	at org.openjdk.jmc.flightrecorder.serializers.json.FlameGraphJsonSerializer.toJson(FlameGraphJsonSerializer.java:89)
	at org.openjdk.jmc.flightrecorder.flameview.views.FlameGraphView$ModelRebuildRunnable.run(FlameGraphView.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ClassNotFoundException: org.openjdk.jmc.flightrecorder.stacktrace.tree.StacktraceTreeModel cannot be found by org.openjdk.jmc.flightrecorder.serializers_8.1.0.qualifier
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:519)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:171)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 5 more
```

Reply:
I could have sworn it was there. Will fix.

Reply:
Nice catch! Thanks @aptmac. :)